R/validate_ensemble_inputs.R

```diff
@@ -28,8 +28,6 @@
 #' @return a list of validated model inputs: `model_outputs` object of class
 #'   `model_output_df`, optional `weights` data frame, and `task_id_cols`
 #'   character vector
-#'
-#' @noRd
```

Comment:
I had read somewhere that this tag was superseded so I removed it but I guess that might have been incorrect. I can put it back in because it wasn't actually throwing any errors